administrator/components/com_media/resources/scripts/components/modals/create-folder-modal.vue

```diff
@@ -1,19 +1,18 @@
 <template>
     <media-modal v-if="$store.state.showCreateFolderModal" :size="'sm'" @close="close()">
-        <h3 slot="header" class="modal-title">{{ translate('COM_MEDIA_CREATE_NEW_FOLDER') }}</h3>
+        <h3 slot="header" class="modal-title">{{ translate('COM_MEDIA_NEW_FOLDER') }}</h3>
         <div slot="body">
             <form class="form" @submit.prevent="save" novalidate>
                 <div class="form-group">
-                    <label for="folder">{{ translate('COM_MEDIA_FOLDER') }}</label>
-                    <input id="folder" class="form-control" placeholder="Folder"
+                    <input id="folder" class="form-control" placeholder="Folder" @focus="$event.target.select()"
```

Comment:
can you check this for closing and reopening too ?
just open the dialog and it should be focused to the input, then close it reopen it and it should be also focused to input

Reply:
@kasvith It working fine.